src/undomanager.js

```diff
@@ -22,16 +22,21 @@ define([
         $('.fd-undo-container').append(UNDO_ALERT);
     }
 
-    function toggleAlert(undoStack) {
+    function toggleAlert(undoStack, vellum) {
```

Comment:
Offlined with @orangejenny. She's going to work on the events suggestion in a separate PR.